### ISSUE TITLE
add support for libretro-style assets in collections

### ISSUE THREAD

**Issue by gtevare** [mmatyas/pegasus-frontend#837]
using linux i have a folder just for assets using this directory structure:

<collection_name>/
└─<asset_type>/
 │ └─<game_title>.png

example:

nintendo - game boy/
├─boxFront/
│  └─Alien 3 (Japan).png
└─screenshot/
│     └─Alien 3 (Japan).png

example metadata file:

`/home/user/.config/pegasus-frontend/metafiles/gameboy.metadata.txt`

```
collection: game boy
command: retroarch -L /home/user/libretro/gambatte_libretro.so {file.name}
directory: /home/user/games/gameboy
extension: gb
<new_all-assets_key>: /home/user/assets/gameboy
```

with this <new_all-assets_key> every game would automatically be paired with same-name assets for each asset type available
could this new key be added for collections?

i hope i explained myself correctly
thanks for making this great software available for everybody

also support for `~/` would be great

**Comment by gtevare** [comment]
yes, only with metadata files inside 'metafiles' folder and 'media' folder in an assignable location

**Comment by gtevare** [comment]
right now you pretty much have to rely on online scraper services since the alternatives seem to be to either

organize asset folders by creating a folder for each game with a folder for each asset type inside, then placing each asset respectively
or
editing metadata files to point to each asset for each asset type for each game

all this would have to be done manually (maybe there's a program?) and its for thousands of games, it's just not feasible

this could be an alternative for people who have their own asset folders or want to download them from other sources (which tend to be Skraper-style organized)

you could keep your 'roms', 'assets' and 'metadata' folders separated
cleaner and simpler, could do with just 5 lines per metadata file, what do you think?

**Comment by mmatyas** [comment]
The idea of a custom location for the media dir does sound technically possible, though I don't know yet how much would it increase the complexity. As a workaround, you could always just move the media/skraper dir somewhere else and create a symlink pointing to that location.

**Comment by gtevare** [comment]
it's not just placing the folder/symlink, 
i can only get the covers working with:

1. metadata.txt inside roms folder
2. assets folder inside roms folder
3. assets folder with one-folder-per-game structure
4. no subfolders (metadata.txt, assets folder and all roms within the first directory level)

'1' and '2' are just annoying
'3' makes offline covers not realistic
'4' really limits organization and flexibility for using other frontends

users wouldn't need to organize their database to fit pegasus,
the idea is to trasfer the complexity from the user to the software,
i hope it's not a lot of complexity though
